rules/cross-platform/initial_access_zoom_meeting_with_no_passcode.toml

```diff
@@ -38,7 +37,7 @@ type = "query"
 
 query = '''
 event.type:creation and event.module:zoom and event.dataset:zoom.webhook and
-  event.action:meeting.created and not zoom.meeting.password:*
+  event.action:meeting.created and not zoom.zoom.meeting.password:*
```

Comment:
does this look right to y'all @peasead @rw-access? This is how it is represented in the dataset for zoom webhook

Reply:
`zoom.zoom.meeting.password`?

Reply:
yea weird - just wanting to verify that it is _actually_ logged that way, or if our schema building logic may somehow be off (for this dataset only, which is unlikely)

Reply:
Based on @peasead observations, it should _not_ need to be `zoom.zoom.*`. The beats schema is built where _all_ fields are double embedded: 

![image](https://user-images.githubusercontent.com/16747370/99578221-fb19df80-2988-11eb-837a-f4e87042a774.png)

I am not sure what would be different about this dataset that would cause it to build different from the others, with the existing logic. Looking into it

Reply:
Dropping conversation from Slack in the event it's needed in this PR.

It is `zoom.meeting.password`.
https://kibana.zocalo.inf.elasticnet.co/s/intelligence-analytics-team/goto/6c0de6ae6bc52012a7186782dd98e855